packages/m/mootable-hashmap.json

```diff
@@ -0,0 +1,34 @@
+{
+  "description": "Full featured, fast, HAMT based HashMaps",
+  "filename": "index.js",
```

Comment:
```suggestion
  "filename": "hashmap.min.js",
```
It looks like the files are called hashmap, not index :)

Reply:
sure is hashmap.min the correct one or just hashmap, I provide both? For now I will make it just hashmap.